src/util/pointInPolygon.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Checks whether a point is inside a polygon
+ * @param {Array} point The point [x1, y1]
+ * @param {Array} vs The vertices [[x1, y1], [x2, y2], ...] of the Polygon
+ * @returns {boolean}
+ */
+export default function pointInPolygon(point, vs) {
```

Comment:
`isPointInPolygon`?

Sometimes we consider a point `{ x, y }`, and sometimes it's `[x1, y1]`. Is there a good reason for the distinction, or is it just the way the cookie's crumbled?

Probably want a short face-to-face on ray-casting algo tomorrow as well.

Reply:
This makes me uncomfortable as well, but I think its a matter of how the cookie crumbled with multiple devs/tools re-using helpers originally only scoped to be used by that particular object. I think in general we should use {x, y} notation ala `cornerstoneMath`, but I'm not sure if scouring the code for this is priority at this time?

> Probably want a short face-to-face on ray-casting algo tomorrow as well.

Hit me up sometime today if you are free, sure we'll have another large bash through all the issue points as I work through them this morning.